### ISSUE TITLE
increase max_page_size to infinity

### ISSUE THREAD

**Issue by batemapf** [18F/tock#163]
https://github.com/18F/tock/blob/master/tock/api/views.py#L21

^^looking for an easy way to dump all timecard data into a single CSV for offiline use. line above seems to be limiting line dump to 500.  @vzvenyach and i were thinking this is set up for something @shawnbot is doing with visualization.  maybe create a timecards_dump.csv/json endpoint that is a clone of the timecards.csv/json endpoint, minus the max_page_size limitation?

**Comment by shawnbot** [comment]
Yeah, this is definitely something we need. I ran into problems with unpaginated data because Django REST Framework wants to serialize and render the whole dataset before sending it, which often causes the request to time out. If I can get streaming requests working, then this should be possible.

**Comment by batemapf** [comment]
Could we set max to like 10K? We are at about 7500 records for the current
FY.  Would get us temporarily functional data dump tool. Or is it anything
much more than 500 causes probs?



On Jun 2, 2015, at 6:08 PM, Shawn Allen <notifications@github.com> wrote:

Yeah, this is definitely something we need. I ran into problems with
unpaginated data because Django REST Framework wants to serialize and
render the whole dataset before sending it, which often causes the request
to time out. If I can get streaming requests working, then this should be
possible.

—
Reply to this email directly or view it on GitHub
<https://github.com/18F/tock/issues/163#issuecomment-108112671>.
